### PR TITLE
Routing

### PR DESCRIPTION
**Pull request by MaxDrozdov**

### CODE REVIEW

File: js/news/news.view.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<div ng-controller="newsCtrl as news">
```
Comment (NE05KY) [inline js/news/news.view.html]: html template in 'js' folder? It's time to rename your folder to 'src'

Comment (MaxDrozdov) [inline js/news/news.view.html]: Done

File: index.html [hunk 2]
```diff
@@ -12,22 +13,29 @@
     <link rel="stylesheet" href="libs/bootstrap/css/bootstrap.min.css">
 
     <script src="js/app.js"></script>
+    <script src="js/appRouting.js"></script>
```
Comment (NE05KY) [inline index.html]: Try to follow one naming convention. All your source files named with dots, but this in camelCase.

Comment (MaxDrozdov) [inline index.html]: Done

**Pull request merged**